### ISSUE TITLE
migrate coreos-cincinnati to the new Fedora OpenShift 4 cluster

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#1181]
coreos-cincinnati is an external facing project and will probably need to be migrated after freeze is over (after Fedora 36 GA).

Here is [the playbook](https://pagure.io/fedora-infra/ansible/blob/main/f/playbooks/openshift-apps/coreos-cincinnati.yml) where the hosts need to be updated to `hosts: os_control:os_control_stg`.

**Comment by lucab** [comment]
Let's start by deploying an additional copy of the services on the new OCP4 cluster: https://pagure.io/fedora-infra/ansible/pull-request/1043.
We will later update DNS entries and steer traffic to this, once all the routes are in place and working.

**Comment by dustymabe** [comment]
We may want to consider moving to F36 at the same time (see checklist from https://github.com/coreos/fedora-coreos-tracker/issues/1101) so we don't have to touch it again in a few weeks.

**Comment by lucab** [comment]
I would rather not throw other things into this cluster migration. But yes, we had to revert to F33 due to cluster limitations: https://pagure.io/fedora-infrastructure/issue/10111. We can move to F36 only after the service is fully switched to the new cluster.

**Comment by lucab** [comment]
The service is now running on the new OCP4 cluster: https://console-openshift-console.apps.ocp.fedoraproject.org/k8s/cluster/projects/coreos-cincinnati. F36 GA date got postponed to next week, so we'll have to wait a bit longer before being able to complete the switchover.

**Comment by lucab** [comment]
F36 got released and fedora-infra is not anymore frozen. We plan to start the service migration on 2022-05-12 09:00 UTC, see https://pagure.io/fedora-infrastructure/issue/10684.

**Comment by lucab** [comment]
Migration finished, all services are up on the new cluster and serving request as of now (2022-05-12 11:15 UTC).

If anybody is observing Cincinnati related troubles (e.g. Zincati logging error messages) after this point, please let us know.
